taxonomy/models.py

```diff
@@ -50,8 +50,7 @@ class Meta:
     @transaction.atomic
     def save(self, *args, **kwargs):
         """Handle slugs."""
-        if self.id is None or self.name != get_object_or_404(
-                Vocabulary, id=self.id).name:
+        if self.id is None:
```

Comment:
It doesn't necessarily create a new entry in the database, see [the Django docs](https://docs.djangoproject.com/en/1.8/ref/models/instances/#how-django-knows-to-update-vs-insert) for information on whether this is done. I'm more concerned why we need to change this behavior at all. We change the slug here on purpose to match the change in the name, and we should keep this behavior. You should update references to the term with the returned `slug` value from the API call.